src/app/public/modules/overlay/overlay.module.ts

```diff
@@ -6,46 +6,19 @@ import {
   NgModule
 } from '@angular/core';
 
-import {
-  RouterModule
-} from '@angular/router';
-
-import {
-  SkyCoreAdapterModule
-} from '../adapter-service/adapter.module';
-
-import {
-  SkyAppWindowRef
-} from '../window/window-ref';
-
-import {
-  SkyOverlayAdapterService
-} from './overlay-adapter.service';
-
 import {
   SkyOverlayComponent
 } from './overlay.component';
 
-import {
-  SkyOverlayService
-} from './overlay.service';
-
 @NgModule({
   imports: [
-    CommonModule,
-    RouterModule,
```

Comment:
Why are we removing `CommonModule` and `RouterModule` ?

Reply:
`CommonModule` should still be imported (I'm seeing it in the source code?), but `RouterModule` isn't needed since it will be imported by the root module, or by each lazy-loaded feature module. We still need the `Router` for the overlay component, however, so I'll add `@angular/router` to the peer dependencies.